samples/notebooks/python/06-memory-and-embeddings.ipynb

```diff
@@ -330,29 +321,12 @@
    ]
   },
   {
-   "cell_type": "markdown",
-   "id": "a4aa0e5d",
-   "metadata": {},
-   "source": [
-    "Let's build a new Kernel."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "f6518515",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "kernel = build_kernel()"
-   ]
-  },
-  {
+   "attachments": {},
    "cell_type": "markdown",
    "id": "75f3ea5e",
    "metadata": {},
    "source": [
-    "Now let's add these files to our VolatileMemoryStore using `SaveReferenceAsync`."
+    "Now let's add these files to our VolatileMemoryStore using SaveReferenceAsync. We'll separate these memories from the chat memories by putting them in a different memory collection."
```

Comment:
nit: the word "memory" is repeated several time. maybe "in a different memory collection" => in a different collection."

also pls word wrap at column ~80/100